lua/neotest-vstest/dotnet_utils.lua

```diff
@@ -284,26 +284,21 @@ function dotnet_utils.get_solution_projects(solution_path)
 
   local solution_dir = vim.fs.dirname(solution_path)
 
-  local projects = {}
+  local projects = dotnet_utils.projects(solution_path)
 
-  if solution_path then
```

Comment:
I removed this entire branch since the only place it is used is from `client.lua` where we are doing this:

`local solution_projects = solution and dotnet_utils.get_solution_projects(solution)`

So it is never called where `solution_path` is nil. We also state that the `@param` solution is `string` and not `string?`